cmd/soroban-cli/src/commands/events.rs

```diff
@@ -226,11 +218,24 @@ impl NetworkRunnable for Cmd {
         client
             .verify_network_passphrase(Some(&network.network_passphrase))
             .await?;
+
+        let contract_ids: Vec<String> = self
+            .contract_ids
+            .iter()
+            .map(|id| {
+                Ok::<String, Error>(
+                    self.locator
+                        .resolve_contract_id(id, &network.network_passphrase)?
+                        .to_string(),
+                )
+            })
+            .collect::<Result<Vec<_>, _>>()?;
```

Comment:
```suggestion
                Ok(
                    self.locator
                        .resolve_contract_id(id, &network.network_passphrase)?
                        .to_string(),
                )
            })
            .collect::<Result<Vec<_>, Error>>()?;
```
You should be able to do it here.

Reply:
Might need `cargo fmt` unfortunately.